doc/GOALS.md

```diff
@@ -1,17 +1,17 @@
 # Goals
 
-The goal of Elektra is to make it trivial to access and specify configuration by an API.
+The goal of Elektra is to make it trivial to access and specify configuration settings by an API.
```

Comment:
Is there a specific reason for this change `configuration` -> `configuration settings`? To me it is fine either way.

Reply:
To make it consistent as described in the glossary. "Configuration" alone is a quite vague and can mean many different things.